mix.exs

```diff
@@ -67,9 +62,20 @@ defmodule Benchee.Mixfile do
       maintainers: ["Tobias Pfeiffer", "Devon Estes"],
       licenses: ["MIT"],
       links: %{
-        "github" => "https://github.com/PragTob/benchee",
+        "Changelog" => "https://hexdocs.pm/benchee/changelog.html",
+        "GitHub" => @source_url,
         "Blog posts" => "https://pragtob.wordpress.com/tag/benchee/"
       }
     ]
   end
+
+  defp docs do
+    [
+      extras: ["CHANGELOG.md", "README.md"],
+      main: "readme",
+      source_url: @source_url,
```

Comment:
all the exdoc docs have source_url on the project level, not here

Reply:
Actually, it can be [applied in both places](https://www.kianmeng.org/2020/11/using-sourceurl-and-homepageurl-with.html). I usually put it under `docs/0` to group it together with `:source_ref` or `:homepage_url`.

Reply:
sadly I get an access denied on your website so can't read the article.

> The owner of this website (www.kianmeng.org) has banned the autonomous system number (ASN) your IP address is in (3209) from accessing this website.

I'll take your word for it though, makes sense that it's definable there I just didn't see it as an example in the docs  and well the docs themselves sadly aren't too good/I never can find a good mix.exs reference for the attributes there.